OracleApplicationExpress/README.md

```diff
@@ -0,0 +1,123 @@
+
+```bash
+    docker run --name oracledb --network=<your network name> oracle/database:12.2.0.1-ee
+```
+
+The database container will be visible within the network by its name passed on with the `--name` option, in the example above **oracledb**.
+Once your database container is up and running and the database available, you can run a new ORDS container.
+
+To run your ORDS Docker image use the **docker run** command as follows:
+
+```bash
+    docker run --name <container name> \
+    --network=<name of your created network> \
+    -p <host port>:8888 \
+    -e ORACLE_HOST=<Your Oracle DB host (default: localhost)> \
+    -e ORACLE_PORT=<Your Oracle DB port (default: 1521)> \
+    -e ORACLE_SERVICE=<your Oracle DB Service name (default: ORCLPDB1)> \
+    -e ORACLE_PWD=<your database SYS password> \
+    -e ORDS_PWD=<your ORDS password> \
+    -e CONTEXT_ROOT=<http context-root to use (default: ords)> \
+    -v [<host mount point>:]/opt/oracle/ords/config/ords \
+    oracle/restdataservices:3.0.12
+    
+    Parameters:
+       --name:            The name of the container (default: auto generated)
+       --network:         The network to use to communicate with databases.
+       -p:                The port mapping of the host port to the container port. 
+                          One port is exposed: 8888
+       -e ORACLE_HOST:    The Oracle Database hostname that ORDS should use (default: localhost)
+                          This should be the name that you gave your Oracle database Docker container, e.g. "oracledb"
+       -e ORACLE_PORT:    The Oracle Database port that ORSD should use (default: 1521)
+       -e ORACLE_SERVICE: The Oracle Database Service name that ORDS should use (default: ORCLPDB1)
+       -e ORACLE_PWD:     The Oracle Database SYS password
+       -e ORDS_PWD:       The ORDS_PUBLIC_USER password
+       -e CONTEXT_ROOT:   (optional) The http context-root that ORDS should use (default: ords)
+       -v /opt/oracle/ords/config/ords
+                          The data volume to use for the ORDS configuration files.
+                          Has to be writable by the Unix "oracle" (uid: 54321) user inside the container!
+                          If omitted the ORDS configuration files will not be persisted over container recreation.
+```
+
+Once the container has been started and ORDS configured you can send REST calls to ORDS.
+
+## Known issues
+
+None
+
+## Support
+
+## License
+
+To download and run ORDS, regardless whether inside or outside a Docker container, you must download the binaries from the Oracle website and accept the license indicated at that page.
+
+All scripts and files hosted in this project and GitHub [docker-images/OracleRestDataServices](./) repository required to build the Docker images are, unless otherwise noted, released under the Universal Permissive License (UPL), Version 1.0.
+
+## Copyright
+
+Copyright (c) 2014-2017 Oracle and/or its affiliates. All rights reserved.
```

Comment:
As before, the copyright date is incorrect and "all rights reserved" should be removed.